### DISCUSSION TITLE
Use space tokens in media tokens?

### DISCUSSION THREAD

**Discussion by ianduvall** [stitchesjs/stitches#868] [Help] [answered]
Is it possible to use space tokens in media tokens when creating a stitches config?

Example:
```javascript
createStitches({
  space: {
    'desktop-min': '1234px',
  },
  media: {
    'desktop-and-up': `(min-width: $space$desktop-min)`,
  },
});
```

Right now I'm just using variables and template literals. Maybe a better question is what tokens can be used in other tokens?

**Comment by peduarte** [comment, marked as answer]
No, you cant do that. `media` is outside of the `theme` config. 

You can use tokens within other tokens in the same scale. https://stitches.dev/docs/tokens#token-aliases
